src/main/java/com/garbagemule/MobArena/waves/WaveParser.java

```diff
@@ -98,6 +98,8 @@ public static Wave parseWave(Arena arena, String name, ConfigurationSection conf
             case BOSS:
                 result = parseBossWave(arena, name, config);
                 break;
+            default:
+                throw new ConfigError("Unknown wave type for wave " + name + " of arena " + arena.configName() + ": " + t);
```

Comment:
I'm not sure why there is a warning about a default branch here, since the current branches are exhaustive. I guess it can't hurt to have it in case a new wave type is introduced, though.